src/Dodo.HttpClient.ResiliencePolicies/RetryPolicy/RetryPolicySettings.cs

```diff
@@ -71,43 +99,54 @@ public static RetryPolicySettings Jitter(int retryCount)
 
 		public static RetryPolicySettings Jitter(int retryCount, TimeSpan medianFirstRetryDelay)
 		{
-			return new RetryPolicySettings(SleepDurationProvider.Jitter(retryCount, medianFirstRetryDelay));
+			return new RetryPolicySettings(retryCount, SleepDurationProvider.Jitter(retryCount, medianFirstRetryDelay));
+		}
+
+		private TimeSpan? getServerWaitDuration(DelegateResult<HttpResponseMessage> response)
+		{
+			var retryAfter = response?.Result?.Headers?.RetryAfter;
+			if (retryAfter == null)
+				return null;
+
+			return retryAfter.Date.HasValue
+				? retryAfter.Date.Value - DateTime.UtcNow
```

Comment:
It is a bit tricky thing. According to MDN (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Retry-After) it can be represented in two formats: http-date or delay in seconds. Here the second case will be represented with Delta proprty. I think we should support this case too.

Reply:
Actualy, Delta was cheked on
`: retryAfter.Delta.GetValueOrDefault(TimeSpan.Zero);`
but its not corretly... return zero delay. 

I reworked it. Check Delta, Date and return null by default. And the code has become more readable.

Reply:
Yeah, my bad. You already deal with it, but I missed it. But anyway it helps to find a bug in this code :)